### ISSUE TITLE
Impossible to run video_room docker container like explained in the README.md

### ISSUE THREAD

**Issue by quentindecock** [membraneframework/membrane_demo#123]
Hi,

In order to test the waters with the Membrane framework I wanted to launch the demo under `webrtc/videoroom/`. I couldn't launch it unfortunately.

Here is what I did:
1. I spun off a brand new Ubuntu 19.03.12 with Docker installed on it on DigitalOcean.
2. I attempted to launch the Docker container as it is insctructed in the README.md file and it did not go through. This is what I got.

```
root@membrane:~# docker run -p 4000:4000 membraneframework/demo_webrtc_videoroom:latest
Unable to find image 'membraneframework/demo_webrtc_videoroom:latest' locally
latest: Pulling from membraneframework/demo_webrtc_videoroom
540db60ca938: Pull complete
7838ce5e8b64: Pull complete
2e30a0b1508e: Pull complete
790e93dd4a52: Pull complete
8d288da1c13a: Pull complete
Digest: sha256:bdd67f6f272432bf029daca353315d62cde1c7585a81c8582c4b2e3dce28c40f
Status: Downloaded newer image for membraneframework/demo_webrtc_videoroom:latest
ERROR! Config provider Config.Reader failed with:
** (ArgumentError) could not fetch environment variable "VIRTUAL_HOST" because it is not set
    (elixir 1.12.0-rc.1) lib/system.ex:699: System.fetch_env!/1
    (stdlib 3.15) erl_eval.erl:685: :erl_eval.do_apply/6
    (stdlib 3.15) erl_eval.erl:446: :erl_eval.expr/5
    (elixir 1.12.0-rc.1) src/elixir.erl:280: :elixir.recur_eval/3
    (elixir 1.12.0-rc.1) src/elixir.erl:265: :elixir.eval_forms/3
    (elixir 1.12.0-rc.1) lib/code.ex:656: Code.eval_string_with_error_handling/3
    (elixir 1.12.0-rc.1) lib/config.ex:258: Config.__eval__!/3
    (elixir 1.12.0-rc.1) lib/config/reader.ex:86: Config.Reader.read!/2

init terminating in do_boot ({,[{Elixir.System,fetch_env!,1,[{_},{_}]},{erl_eval,do_apply,6,[{_},{_}]},{erl_eval,expr,5,[{_},{_}]},{elixir,recur_eval,3,[{_},{_}]},{elixir,eval_forms,3,[{_},{_}]},{Elix
{"init terminating in do_boot",{#{'__exception__'=>true,'__struct__'=>'Elixir.ArgumentError',message=><<"could not fetch environment variable \"VIRTUAL_HOST\" because it is not set">>},[{'Elixir.System','fetch_env!',1,[{file,"lib/system.ex"},{line,699}]},{erl_eval,do_apply,6,[{file,"erl_eval.erl"},{line,685}]},{erl_eval,expr,5,[{file,"erl_eval.erl"},{line,446}]},{elixir,recur_eval,3,[{file,"src/elixir.erl"},{line,280}]},{elixir,eval_forms,3,[{file,"src/elixir.erl"},{line,265}]},{'Elixir.Code',eval_string_with_error_handling,3,[{file,"lib/code.ex"},{line,656}]},{'Elixir.Config','__eval__!',3,[{file,"lib/config.ex"},{line,258}]},{'Elixir.Config.Reader','read!',2,[{file,"lib/config/reader.ex"},{line,86}]}]}}

Crash dump is being written to: erl_crash.dump...done
root@membrane:~#
```

I did setup the `VIRTUAL_HOST` variable with a preceding `export VIRTUAL_HOST=<server-ip>` but it didn't solve the problem.

I believe you intended to make this work out of the box, though it is not at the moment. Is there anything obvious I am missing ? Is there anything I can do to help fix this ?

Best regards,

Quentin

**Comment by mickel8** [comment]
Hi @quentindecock, we are fixing our docker images as we did some changes to the `membrnae_demo` (take a look at #121 ). At this moment, try to build docker image on your own and let us know if this works for you :wink:

**Comment by quentindecock** [comment]
Hello @mickel8 !

Thanks a lot for your fast answer. I wasn't expecting that :)

Building from the Dockerfile fails too. This is what I get when running  `docker build  -t membrane_videoroom .` (I have removed steps 1 to 10 which build just fine.

```
root@membrane:~/membrane_demo/webrtc/videoroom# docker build  -t membrane_videoroom .
[step 1 to 10 removed]
Step 11/26 : COPY priv priv
COPY failed: file not found in build context or excluded by .dockerignore: stat priv: file does not exist
```

I tried to comment the `COPY priv priv` in the Dockerfile, but then the following step fails. Maybe the `priv` folder was removed when it shouldn't have.

Please let me know if there is any way I can help.

Have a good day man!

**Comment by mickel8** [comment]
Hmm that sounds strange. Please let me know if https://github.com/membraneframework/membrane_demo/pull/125 works for you. It works for me - I am able to build and run docker image without errors

**Comment by mickel8** [comment]
Hi @quentindecock, after some fixes in #125 everything works well for me so I am closing this issue. Feel free to reopen if you encounter any further problems :slightly_smiling_face:

**Comment by quentindecock** [comment]
Hey @mickel8 , thanks for the udpate. I'll take a look at it and will update you. Cheers !
